### PR TITLE
Fixes and template improvements for motion comments.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Fixed erros while creating motion as anonymous user.

### CODE REVIEW

File: openslides/motions/views.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def create(self, request, *args, **kwargs):
         if (not request.user.has_perm('motions.can_see_and_manage_comments') and
                 request.data.get('comments')):
             # Some users are not allowed to send comments data.
-            self.permission_denied(request)
+            del request.data['comments']
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Why this? I think we should hide the field in the form (See MotionComment.getFormFields in site.js I think). Then the user should not submit comments if he is not authorized.

File: openslides/users/static/js/users/site.js [hunk 2]
```diff
@@ -1311,6 +1311,7 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
         gettext('Can create motions');
         gettext('Can support motions');
         gettext('Can manage motions');
+        gettext('Can see and manage motions');
```
Comment (normanjaeckel) [inline openslides/users/static/js/users/site.js]: This string does not exist on serverside. Its something with comments I think.

**Comment by normanjaeckel** [comment]
Some remarks.

**Comment by emanuelschuetze** [comment]
All done. Rebased.

**Pull request merged**